### ISSUE TITLE
When using tab to navigate flow on login is incorrect

### ISSUE THREAD

**Issue by hashraf88** [Alfresco/alfresco-ng2-components#365]
When using tab to tab through the content the sequence should be username > password > sign in button but is username > password > top left menu option

[login.mp4.zip](https://github.com/Alfresco/alfresco-ng2-components/files/349511/login.mp4.zip)

n.b. Enable voice over on mac with command and F5

**Comment by hashraf88** [comment]
This is also highlighted using the Wave tool to, due to username and password containing positive tab index and so causes confusion when using tab to navigate. A possible solution is to either have all interactable elements with tab index or none. 


<input id="username" class="mdl-textfield__input ng-dirty ng-valid ng-touched" type="text" _ngcontent-imb-20="" data-automation-id="username" ngcontrol="username" **tabindex="1"** ng-reflect-name="username">
</div>

<input id="username" class="mdl-textfield__input ng-dirty ng-valid ng-touched" type="text" _ngcontent-imb-20="" data-automation-id="username" ngcontrol="username" **tabindex="1"** ng-reflect-name="username">
</div>
